src/audio_call.c

```diff
@@ -919,25 +841,39 @@ void cmd_bitrate(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)
 
     char *error_str;
 
-    if (argc != 1) {
-        error_str = "Must have value!";
+    Call *call = &CallControl.calls[self->num];
+
+    if (call->status != cs_Active) {
+        error_str = "Must be in a call";
         goto on_error;
     }
 
-    if (self->is_call == false) {
-        error_str = "Must be in a call";
+    if (argc == 0) {
+        line_info_add(self, NULL, NULL, NULL, SYS_MSG, 0, 0,
+                      "Current audio encoding bitrate: %u", call->audio_bit_rate);
+        return;
+    }
+
+    if (argc > 1) {
+        error_str = "Too many arguments.";
         goto on_error;
     }
 
-    const uint32_t bitrate = strtol(argv[1], NULL, 10);
+    char *end;
+    const long int bit_rate = strtol(argv[1], &end, 10);
+
+    if (*end || bit_rate < 0) {
```

Comment:
We may also want to avoid the bit rate exceeding UINT32_MAX (although this will give a warning if you compile for ILP32).